tests/test_layout.py

```diff
@@ -10,8 +10,6 @@
 class TestLayout:
 
     @pytest.mark.nondestructive
-    @pytest.mark.xfail("'mozilla.com' in config.getvalue('base_url')",
-                       reason="https://bugzilla.mozilla.org/show_bug.cgi?id=1201622")
```

Comment:
The bug is about Metro Firefox, but there's no mention of it in the test. I'm guessing this has been xpassing for a while. Anyway, yay for removing an xfail! :grinning: